transforms/utils.js

```diff
@@ -0,0 +1,109 @@
+// The following utils are borrowed from @simonihmig's
+// https://github.com/simonihmig/ember-test-helpers-codemod/blob/master/transforms/utils.js
+
+let _statementsToImport = new Set();
+
+/**
+ * Adds (one or more) named imports to a private import statement collection.
+ * To write the import statements to a file, use writeImportStatements.
+ *
+ * @param j
+ * @param {array} imports
+ */
+function addImportStatement(imports) {
+  imports.forEach(method => _statementsToImport.add(method));
+}
+
+/**
+ * Adds all collected named imports to an (existing or to be created) `import { namedImport } from 'htmlbars-inline-precompile';
+ * To add named imports to the collection, use addImportStatement
+ *
+ * @param j
+ * @param root
+ */
+function writeImportStatements(j, root) {
+  if (_statementsToImport.size > 0) {
+    let body = root.get().value.program.body;
+
+    const inlineImportStatement = root.find(j.ImportDeclaration, {
+      source: {
+        value: 'htmlbars-inline-precompile',
+      },
+    });
+
+    const emberCliImportStatement = root.find(j.ImportDeclaration, {
+      source: {
+        value: 'ember-cli-htmlbars',
+      },
+    });
+
+    if (inlineImportStatement.length !== 0) {
+      inlineImportStatement.remove();
+    }
+
+    if (emberCliImportStatement.length === 0) {
+      const importStatement = createImportStatement(
+        j,
+        'ember-cli-htmlbars',
+        'default',
```

Comment:
This seems surprising, why do we pass `default` here for `imported`? It seems like we actually want `createImportStatement` to take only one argument for `imported`, and it should be probably be a Map (instead of an array/set).